game/script-jokes.rpy

```diff
@@ -881,3 +911,78 @@ label joke_pointless_pencil:
     extend 1fsqsm " It's {i}pointless{/i}."
 
     return
+
+label joke_know_the_drill:
+    n 1ulraj "You know,{w=0.2} [player]..."
+    n 1nsqsl "I always used to get annoyed at people doing construction work.{w=0.75}{nw}"
+    extend 1fnmem " {i}Especially{/i} on the weekends!"
+    n 1fllfl "Like...{w=1}{nw}"
+    extend 1fcsbo " I get they have a job to do.{w=0.75}{nw}"
+    extend 1fsran " But do they seriously have to start so {i}early{/i}?!{w=0.75}{nw}"
+    extend 1fcsan " Yeesh!"
+    n 1cslsl "..."
+    n 1cslaj "But...{w=1}{nw}"
+    extend 1cllca " you do get used to all the noise after a while,{w=0.2} I suppose."
+    n 1cnmss "I guess eventually you just...{w=1}{nw}"
+    extend 1fsqbg " {i}know the drill{/i},{w=0.75}{nw}"
+    extend 1nchgn " right?"
+
+    return
+
+label joke_can_do_attitude:
+    n 1fcsbg "Let's see how well you can {i}process{/i} this one,{w=0.2} [player].{w=0.75}{nw}"
+    extend 1fsqsm " Ehehe."
+    n 1fcsbs "So!{w=0.75}{nw}"
+    extend 1tsqss " Ever wondered what it takes to land a job in a cannery?"
+    n 1tsqsm "..."
+    n 1fsqsm "No?{w=0.75}{nw}"
+    extend 1fcsbs " Come on,{w=0.2} [player]!{w=0.75}{nw}"
+    extend 1fcssmesm " Isn't it obvious?"
+    n 1fcsbg "...You just need a{w=0.5}{nw}"
+    extend 1fsqbg " {i}can{/i}-do{w=1}{nw}"
+    extend 1uchgn " attitude!"
+
+    return
+
+label joke_out_of_ctrl:
+    n 1fllbo "I gotta say,{w=0.2} [player].{w=0.75}{nw}"
+    extend 1fsqfl " I'm starting to get {i}really{/i} sick of all these stories about shortages."
+    n 1fnmem "Seriously -{w=0.5}{nw}"
+    extend 1fcsgs " it's ridiculous!{w=0.75}{nw}"
+    extend 1flrfl " Like why is it so hard to just order stuff in,{w=0.2} all of a sudden?"
+    n 1fcsgs "I mean,{w=0.5}{nw}"
+    extend 1fcspo " even just the other day I heard about places running out of parts for keyboards!"
+    n 1fcsss "...Heh."
+    n 1fcstr "I guess their management must really be{w=0.5}{nw}"
```

Comment:
```suggestion
    n 1fcsgs "I mean,{w=0.5}{nw}"
    extend 1fslpo " even just the other day I heard about places running out of parts for keyboards!"
    n 1fcsss "...Heh."
    n 1fnmtr "I guess their management must really be{w=0.5}{nw}"
```
Overall Natsuki has her eyes closed a lot in within this joke, but here she has them straight closed for 4 lines! I think it would be better to have more variety in that case, how else would she be able to read that joke?

Reply:
That's fair - honestly I have a tendency to use the closed-eye expressions since they work really well with a lot of mouth types, so I'll need to watch out for this more.